### ISSUE TITLE
SystemError: <class 'wx._core.Size'> returned a result with an exception set

### ISSUE THREAD

**Issue by boergens** [DeepLabCut/DeepLabCut#1889]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

When trying to start deeplabcut, it crashed with the above message. At first I thought I was seeing #682 but I am on wxPython==4.0.7.post2

### Operating System

Ubuntu 22.04

### DeepLabCut version

2.2.1.1

### DeepLabCut mode

multi animal

### Device type

GeForce RTX 2070 Mobile

### Relevant log output

```shell
(env)kevin@ubuntu:~/deeplabcut_2022$ python3 -m deeplabcut
Loading DLC 2.2.1.1...
/home/kevin/deeplabcut_2022/env/lib/python3.10/site-packages/statsmodels/compat/pandas.py:65: FutureWarning: pandas.Int64Index is deprecated and will be removed from pandas in a future version. Use pandas.Index with the appropriate dtype instead.
  from pandas import Int64Index as NumericIndex
Starting GUI...
TypeError: 'float' object cannot be interpreted as an integer

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/kevin/deeplabcut_2022/env/lib/python3.10/site-packages/deeplabcut/__main__.py", line 22, in <module>
    deeplabcut.launch_dlc()
  File "/home/kevin/deeplabcut_2022/env/lib/python3.10/site-packages/deeplabcut/gui/launch_script.py", line 49, in launch_dlc
    frame = MainFrame().Show()
  File "/home/kevin/deeplabcut_2022/env/lib/python3.10/site-packages/deeplabcut/gui/launch_script.py", line 23, in __init__
    super(MainFrame, self).__init__("DeepLabCut")
  File "/home/kevin/deeplabcut_2022/env/lib/python3.10/site-packages/deeplabcut/gui/widgets.py", line 72, in __init__
    size=wx.Size(self.gui_size),
SystemError: <class 'wx._core.Size'> returned a result with an exception set
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by jeylau** [comment]
@boergens, I've pushed a small fix in #1893 but I am not 100% certain it addresses your issue, as I couldn't reproduce the error. Please let me know if it does indeed solve your problem :)

**Comment by boergens** [comment]
@jeylau Yes, this fixed it (I technically didn't try in master, only in release 2.2.1.1)
Thanks a lot!
